gdal/gcore/gdaldataset.cpp

```diff
@@ -4578,6 +4578,30 @@ OGRLayerH GDALDatasetExecuteSQL( GDALDatasetH hDS,
             pszDialect));
 }
 
+
+/************************************************************************/
+/*                        GDALDatasetAbortSQL()                       */
+/************************************************************************/
+
+/**
+ \brief Abort any SQL statement running in the data store.
+
+ This method is the same as the C++ method GDALDataset::AbortSQL()
+
+ @since GDAL 3.1.4
+
+ @param hDS the dataset handle.
+
+*/
+
+void GDALDatasetAbortSQL( GDALDatasetH hDS )
```

Comment:
wondering if we shouldn't make it return a CPLErr ?
The base dummy implementation would return a CE_Failure

Reply:
Yes, I alto thought about that, it's a good idea.